lib/components/chat_history/twitch/poll_indicator.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:provider/provider.dart';
+import 'package:rtchat/models/messages/twitch/event.dart';
+import 'package:rtchat/models/style.dart';
+
+class PollIndicatorWidget extends StatelessWidget {
+  final PollChoiceData data;
+  final bool isCompleted;
+  final int maxVotes;
+  const PollIndicatorWidget({
+    Key? key,
+    required this.data,
+    required this.isCompleted,
+    required this.maxVotes,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Consumer<StyleModel>(builder: (context, styleModel, child) {
+      var baseStyle = Theme.of(context)
+          .textTheme
+          .bodyText2!
+          .copyWith(fontSize: styleModel.fontSize);
+      // winner poll gets a trophy icon
+      return Padding(
+          padding: const EdgeInsets.only(bottom: 8),
+          child: Stack(alignment: AlignmentDirectional.center, children: [
+            SizedBox(
+              height: 37,
+              child: LinearProgressIndicator(
+                value: data.percentage,
+              ),
+            ),
+            if (isCompleted && data.votes >= maxVotes) ...[
+              const Padding(
+                padding: EdgeInsets.only(left: 8),
+                child: Align(
+                  child: Icon(
+                    Icons.emoji_events_outlined,
+                    size: 32,
+                  ),
+                  alignment: Alignment.centerLeft,
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.only(left: 48),
+                child: Align(
+                  child: Text(
+                    data.title,
+                    style: baseStyle,
+                  ),
+                  alignment: Alignment.centerLeft,
+                ),
+              )
+            ],
+            if (!isCompleted || data.votes < maxVotes) ...[
```

Comment:
`else` instead of the inverted conditional.

Also the spread operator can be omitted since there's only one child.

Reply:
It didn't let me do the `else`. and `...[]` also seems necessary. IDE wasn't happy with either